server/controllers/updateIsSitter.js

```diff
@@ -0,0 +1,33 @@
+const Profile = require("../models/Profile");
+
+const getSitterStatus = async (_id) => {
+  const response = await Profile.find({ _id }, "isSitter", (err, result) => {
+    if (err) throw new Error(err);
+    return result;
+  });
+  return response;
+};
+
+const changeIsSitter = async (_id, value, res) => {
+  await Profile.findByIdAndUpdate(String(_id), { isSitter: value }, (err, result) => {
+    if (err) {
+      res.status(500).send(`A problem occurred while changing sitter status. ${err}`);
+    } else res.status(200).send(`Sitter status changed. ${result}`);
+  });
+};
+
+// @route PATCH /profile/sitter
+// @desc become a sitter or not
+// @access Private
+exports.updateIsSitter = async (req, res) => {
+  const { _id } = req.query;
```

Comment:
Consider sending data in the request body, query parameters are optional and mainly used when we want to query throught the database

Reply:
I fixed the editProfile function in the profile controller, and now fetch uses a post method to send data to the profile controller. So this controller was deleted.